subprojects/dependency-management/src/main/java/org/gradle/internal/component/model/ConfigurationMetadata.java

```diff
@@ -87,4 +88,6 @@
      * (For external module components, we just instantiate a new artifact metadata).
      */
     ComponentArtifactMetadata artifact(IvyArtifactName artifact);
+
+    CapabilitiesMetadata getCapabilitiesMetadata();
```

Comment:
Nit: to be consistent with other methods, this should be called simply `getCapabilities()`